src/Uno.UI/Microsoft/UI/Xaml/Controls/Common/ColorConversion.cs

```diff
@@ -0,0 +1,346 @@
+﻿using System;
+using System.Globalization;
+using Windows.UI;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	internal static class ColorConversion
+	{
+		public static UInt64? TryParseInt(string s)
+		{
+			return TryParseInt(s, 10 /* numBase */);
+		}
+
+		public static UInt64? TryParseInt(string str, int numBase)
+		{
+			// If we have a zero-length string, then we can immediately know
+			// that this is not a valid integer.
+			if (string.IsNullOrEmpty(str))
+			{
+				return null;
+			}
+
+			// Uno Doc: Using C#/.net methods instead, only works for base 10 or 16
+			switch (numBase)
+			{
+				case 10:
+				{
+					if (UInt64.TryParse(str, out UInt64 result))
+					{
+						return result;
+					}
+					break;
+				}
+
+				case 16:
+				{
+					if (UInt64.TryParse(str, NumberStyles.HexNumber, CultureInfo.InvariantCulture, out UInt64 result))
+					{
+						return result;
+					}
+					break;
+				}
+			}
+
+			// Uno Doc: C++ methodology does not easily convert to C#
+			/*wchar_t *end;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)